frontend/src/lib/lemon-ui/LemonTree/LemonTree.stories.tsx

```diff
@@ -29,6 +29,7 @@ const meta: Meta<typeof LemonTree> = {
             {
                 id: 'gt_7d8f9',
                 name: 'Growth team',
+                type: 'folder',
```

Comment:
Shouldn't the existence of `children`, even if `children:[]` effectively do the same as `type: 'file'/'folder'`? If so, no need to have 2x the same info.